daemon/src/rollover_maker.rs

```diff
@@ -98,7 +98,7 @@ impl Actor {
         let mut conn = self.db.acquire().await?;
         let cfd = load_cfd(self.proposal.order_id, &mut conn).await?;
 
-        let event = cfd.receive_rollover_proposal(self.proposal.clone())?;
```

Comment:
I think this highlights how useless the data in `RolloverProposal` currently is :joy:

Reply:
Yep, we want to get rid of the timestamp. The order id is necessary at some level, but maybe the `RolloverProposal` itself can omit it?

Reply:
Depends on what level. When sending from taker to maker we must include the id. 
But within the context for a rollover actor we know the id anyway...

Reply:
- On the wire we certainly need the ID.
- Once we loaded the CFD from the DB, a parameter-less command should be enough to move forward.